app/models/driver.rb

```diff
@@ -3,6 +3,6 @@ class Driver < ApplicationRecord
   validates :license_number, presence: true
   validates :mobile_number, presence: true, phone: true
   validates :email_address, presence: true, email: true
-  validates :age, presence: true
-  validates :status, presence: true
+  validates :age, presence: true, numericality: { greater_than: 0}
+  validates :status, presence: true, inclusion: { in: [ true, false ] }
```

Comment:
i think when you run tests this should shout some warnings at you

Reply:
the age validation doesnt work but the status one does

Reply:
ah the status one will give warnings when tested with shoulda matchers actually.
since ensuring a boolean is true or false is redundant.

Reply:
so should i remove the inclusion bit for the status?

Reply:
nah lets leave it for now

Reply:
but lets add a test for the `inclusion:` part